SolutionInfo.cs

```diff
@@ -18,6 +18,6 @@
 [assembly: AssemblyProduct("https://dnncommunity.org")]
 [assembly: AssemblyCopyright("DNN Platform is copyright 2002-2019 by .NET Foundation. All Rights Reserved.")]
 [assembly: AssemblyTrademark("DNN")]
-[assembly: AssemblyVersion("9.4.4")]
-[assembly: AssemblyFileVersion("9.4.4.5")]
-[assembly: AssemblyInformationalVersion("9.4.4.5 Custom build")]
+[assembly: AssemblyVersion("9.6.0")]
+[assembly: AssemblyFileVersion("9.6.0.5")]
+[assembly: AssemblyInformationalVersion("9.6.0-rc.3+Branch.release-9.6.0.Sha.600e5a8e59d14c432645384f6cdeb6c97fadff36")]

```

Comment:
I don't think it makes sense for these to point to a specific build, how about this?
```suggestion
[assembly: AssemblyFileVersion("9.6.0.0")]
[assembly: AssemblyInformationalVersion("9.6.0.0 Custom build")]

```

Reply:
Yeah, this was done using the GitVersion we use. I can also edit this by hand. Hopefully for the last time.